### ISSUE TITLE
Updating taphome light took longer than the scheduled update interval

### ISSUE THREAD

**Issue by martindybal** [martindybal/taphome-homeassistant#37]
Sometimes happens to me that update entity state take a long time (longer than 10 seconds). It can be caused by my network. Information about this issue can be found in HA log http://127.0.0.1:8123/config/logs

**Comment by polcape** [comment]
Hi @martindybal,

it happen also to my infastructure. Maybe the polling to request the state are too frequent and HA can't manage the answer?

**Comment by martindybal** [comment]
In my case response from cloudapi takes more than 10 seconds. Home Assistant ask for actual state of each device every 10 seconds and fails if update took longer than that. 

We can't fix this on our site, maybe we can use longer polling interval but I don't like it. It doesn't happen so often. I would leave it as is and watch it for now.

**Comment by martindybal** [comment]
Related with #4

**Comment by martindybal** [comment]
```
2020-12-23 04:34:26 WARNING (MainThread) [homeassistant.components.cover] Updating taphome cover took longer than the scheduled update interval 0:00:10
2020-12-23 04:34:26 WARNING (MainThread) [homeassistant.components.light] Updating taphome light took longer than the scheduled update interval 0:00:10
2020-12-23 04:34:26 WARNING (MainThread) [homeassistant.helpers.entity] Update of cover.blinds_kitchen_south is taking over 10 seconds
2020-12-23 04:34:26 WARNING (MainThread) [homeassistant.helpers.entity] Update of cover.roleta is taking over 10 seconds
2020-12-23 04:34:26 WARNING (MainThread) [homeassistant.helpers.entity] Update of cover.zaluzie_90deg is taking over 10 seconds
2020-12-23 04:34:26 WARNING (MainThread) [homeassistant.helpers.entity] Update of cover.zaluzie_180deg is taking over 10 seconds
2020-12-23 04:34:26 WARNING (MainThread) [homeassistant.helpers.entity] Update of light.rgb_1 is taking over 10 seconds
2020-12-23 04:34:26 WARNING (MainThread) [homeassistant.helpers.entity] Update of light.svetlo_rgb is taking over 10 seconds
2020-12-23 04:34:26 WARNING (MainThread) [homeassistant.helpers.entity] Update of light.svetlo is taking over 10 seconds
2020-12-23 05:01:46 WARNING (MainThread) [homeassistant.components.cover] Updating taphome cover took longer than the scheduled update interval 0:00:10
2020-12-23 05:01:46 WARNING (MainThread) [homeassistant.components.light] Updating taphome light took longer than the scheduled update interval 0:00:10
2020-12-23 05:01:46 WARNING (MainThread) [homeassistant.helpers.entity] Update of cover.zaluzie_90deg is taking over 10 seconds
2020-12-23 05:01:46 WARNING (MainThread) [homeassistant.helpers.entity] Update of light.svetlo_rgb is taking over 10 seconds
```

**Comment by martindybal** [comment]
Can't be fixed on our side. I hope it will be solved by #4 or with local api support
